spack.yaml

```diff
@@ -0,0 +1,9 @@
+spack:
+  specs:
+  - cmake@3.25.1
+  - mpich@4.0.2~fortran +verbs +argobots ^libfabric@1.16.1 fabrics=mlx,rxd,rxm,shm,sockets,tcp,udp,verbs
```

Comment:
If you choose mpich+verbs, will it configure mpi to use verbs and not sockets, even if both are supported on a machine?